db-init/src/main/resources/database/migrations/V4.9__Add_Sufficient_Evidence_Flag.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE assessment_result ADD COLUMN sufficient_evidence_flag BOOLEAN;
```

Comment:
I want to confirm that `sufficient_evidence_flag` can be null, so that we can treat values True, False, and Null as distinct values like so:
<img width="327" alt="image" src="https://user-images.githubusercontent.com/55255674/219511784-a05a7809-254c-457e-8075-9208279024a1.png">

Reply:
As per below comments, it absolutely can be null. Resolving this comment. Will resolve other comment once unit test is in.